src/requireLocation.ts

```diff
@@ -0,0 +1,10 @@
+export const requireLocation = (store: any) => {
+  return (nextState: any, replace: any) => {
+    if (!store.getState().user.location) {
+      replace({
+        pathname: '/',
+        state: { nextPathname: nextState.location.pathname }
```

Comment:
right, but the "state: { nextPathname: nextState.location.pathname }" part, what is nextPathname used for?